fbchat/client.py

```diff
@@ -938,7 +938,7 @@ def sendEmoji(self, emoji=None, size=EmojiSize.SMALL, thread_id=None, thread_typ
 
         return self._doSendRequest(data)
 
-    def _uploadImage(self, image_path, data, mimetype):
+    def _uploadImage(self, image_path, data, mimetype, is_gif=False):
```

Comment:
In here, we are getting the `mimetype` of the image, perhaps we can use that to differentiate between gifs/other images instead?

Reply:
Thanks. I tried that and it works! Already changed to that.